docs/queues.txt

```diff
@@ -9,24 +9,27 @@ Queues
    :values: tutorial
 
 .. meta::
-   :keywords: php framework, odm, code example
+   :keywords: php framework, odm, code example, jobs
 
-If you want to use MongoDB as your database backend for Laravel Queue, change
-the driver in ``config/queue.php``:
+To use MongoDB as your database for Laravel Queue, change
+the driver in your application's ``config/queue.php`` file:
 
 .. code-block:: php
 
    'connections' => [
        'database' => [
            'driver' => 'mongodb',
-           // You can also specify your jobs specific database created on config/database.php
+           // You can also specify your jobs-specific database created on config/database.php
            'connection' => 'mongodb',
            'collection' => 'jobs',
            'queue' => 'default',
            'retry_after' => 60,
```

Comment:
In https://github.com/mongodb/laravel-mongodb/pull/2985#discussion_r1622733967 we decided with @jmikola to comment optional settings.
```suggestion
           // 'collection' => 'jobs',
           // 'queue' => 'default',
           // 'retry_after' => 60,
```